db/migrate_cards/20160105154503_import_tweaks_and_bugfixes.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportTweaksAndBugfixes < Card::Migration
+  def up
+    import_json "tweaks_and_bugfixes.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.